code/modules/antagonists/voidwalker/voidwalker_loot.dm

```diff
@@ -37,3 +40,4 @@
 	uses--
 	if(uses <= 0 )
 		icon_state = drained_icon_state
+		light_on = 0
```

Comment:
should be using `TRUE` and `FALSE` here if this is treated just as a boolean

Reply:
yep